index.js

```diff
@@ -16,9 +16,20 @@ const AWS = require("aws-sdk");
     const prefix = core.getInput("prefix");
     const Path = core.getInput("path", { required: true });
     const WithDecryption = core.getInput("decrypt") === "true";
-    const { Parameters } = await ssm.getParametersByPath({ Path, WithDecryption }).promise();
 
-    Parameters.forEach(({ Name, Value, Type }) => {
+    let parameters = [];
+
+    let { NextToken, Parameters } = await ssm.getParametersByPath(params).promise();
+
+    parameters = [...Parameters];
+    while (NextToken) {
+      params.NextToken = NextToken;
+      results = await ssm.getParametersByPath(params).promise();
+      parameters = [...parameters, ...results.Parameters];
+      NextToken = results.NextToken;
+    }
+
+    parameters.forEach(({ Name, Value, Type }) => {
```

Comment:
Nice! 👌 Thanks for adding this; totally forgot about pagination here.

Looks like there's a few vars that just need to be declared first; also, we should just be able to push the new params onto the original array, right?
```suggestion
    const options = { Path, WithDecryption };
    let { NextToken, Parameters } = await ssm.getParametersByPath(options).promise();
    
    while (NextToken) {
      options.NextToken = NextToken;
      const results = await ssm.getParametersByPath(options).promise();
      Parameters.push(results.Parameters);
      NextToken = results.NextToken;
    }

    Parameters.forEach(({ Name, Value, Type }) => {
```